lib/triplex.ex

```diff
@@ -157,18 +181,23 @@ defmodule Triplex do
   `new_tenant`.
   """
   def rename(old_tenant, new_tenant, repo \\ config().repo) do
-    if reserved_tenant?(new_tenant) do
+    cond do
+     reserved_tenant?(new_tenant) ->
       {:error, reserved_message(new_tenant)}
-    else
+    repo.__adapter__ == Ecto.Adapters.MySQL ->
+      {:error, "you cannot rename tenants in a MySQL database. You will have to drop the tenant and recreate it with a different name"}
```

Comment:
Well, to drop the tenant and create another is not a good option, since the user will lose all his tenant data. Let's remove this part and stay only with: `"you cannot rename tenants in a MySQL database"`

Reply:
Good point, changed.